lib/hooks/create/project.js

```diff
@@ -43,6 +44,17 @@ module.exports = settings => {
             }
           });
 
+      case 'update-issue-date':
+        return Promise.resolve()
+          .then(() => Profile.query().findById(changedBy))
+          .then(profile => {
+            if (!profile.asruUser || !profile.asruLicensing) {
+              throw new UnauthorisedError('Only ASRU licensing officers can change the granted date');
+            }
```

Comment:
I'm not sure this check is actually good. There's a permissions check that looks for permissions on the task `project.updateIssueDate` upstream, and then this does a separate - potentially conflicting - check.

I think I'd prefer that if we were going to check permissions here (which I have no particular concern with as a concept) then it was via a permissions lookup rather than re-implementing the same check in a different way.